lib/pdf/core/embedded_files.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module PDF
+  module Core
+    module EmbeddedFiles #:nodoc:
+      # The maximum number of children to fit into a single node in the
+      # EmbeddedFiles tree.
+      NAME_TREE_CHILDREN_LIMIT = 20 #:nodoc:
```

Comment:
I used that number because it's the limit used for the name tree of the Destinations. I think this limit can vary but I just needed one to create the name tree. It will just create more or fewer nodes.